src/ExpenseTracker.Common/Constants/Constant.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ExpenseTracker.Common.Constants
+{
+    public class Constant
```

Comment:
@BirajMainali Maybe we should rename it to `StatusConstants` or something. Its more revealing that way.